privacyidea/static/components/token/views/token.enrolled.hotp.html

```diff
@@ -29,7 +32,7 @@
         </button>
         </div>
     </div>
-    <div class="col-sm-4 pull-right" ng-show="form['2stepinit'] && enrolledToken.rollout_state === 'clientwait'">
+    <div class="col-sm-6" ng-show="form['2stepinit'] && enrolledToken.rollout_state === 'clientwait'">
```

Comment:
Why is it necessary to increase the image size from 4/12 to 6/12?

Reply:
Half of the space for the QR code, and half of the space for the 2 step info.

Reply:
OK, now I understand. Only 10 of 12 elements were used.